### DISCUSSION TITLE
Welcome to Pieces for Developers Support!

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#1] [Announcements]
Welcome to the Pieces for Developers support discussion page! We are here to help - and so is the rest of our community!

Pieces is a new product that is growing at light speed, and we encourage our community to be active participants in the development so that it is tailored exactly for our users. Please take a look at our documentation [on our website](https://docs.pieces.app/), join our [Discord community](https://discord.gg/getpieces), and check out our Open Source repo.

Use Issues to search problems and solutions, and submit an issue if you run into a problem you can't solve. 

For everything else - join us in Discussions!

**Comment by DarshanBanait** [comment]
I had a brilliant idea: can we integrate some awesome automatic time tracking and activity categorization and awesome dashboards for your time spent on various activities to let the users get some insight into their time spent online?
This would definitely add more value to Pieces as a one-stop app for everything.

**Reply by rosie-at-pieces** [reply]
Love this idea @DarshanBanait !!

**Comment by dravv-alt** [comment]
Subject: Issues with Persistent Background Services & Startup Behavior – Pieces OS

Hi Pieces Team,

I hope you're doing well. I recently installed Pieces OS for a course project and allowed local access as prompted. However, I’ve noticed some issues that I’m hoping you can help me with.

Despite not using the app for several months and closing it from the system, I’ve observed that:

The app continues to run background services and attempts to launch at system startup, including entries like PsfLauncher64.exe.

I’m unable to fully disable the app via Task Manager or the Startup settings.

There are still service-related files and folders remaining in local app directories.

The app seems to try to access the camera occasionally, which has raised some privacy concerns.


I would appreciate it if you could provide:

1. A clear guide on how to properly uninstall and clean up all associated files, registry keys, and startup tasks.


2. Information about any data that might have been stored or transmitted while the app was running.


3. Clarification on how this behavior aligns with your privacy policy and user consent process.



I’d really appreciate your help in resolving these issues. Thanks in advance for your time and support!

Best regards

**Reply by FnBrian79** [reply]
Those are very intriguing issues I'm interested to see how they respond

**Reply by mack-at-pieces** [reply]
Hi @dravv-alt -

Thank you for reaching out and for taking the time to share your experience in such detail, we really appreciate it.

We’d love to better understand what you’re seeing and make sure everything is working as expected on your system. Would you be open to hopping on a quick call with a few members of our team? We can walk through your setup together, take a closer look at the behavior you described, and answer any questions you have in real time.

If that sounds good, you can schedule a time that's convenient for you using this calendar link: https://calendar.google.com/calendar/appointments/schedules/AcZssZ3bJHPPSJJHom2WwsPf5KcNlu-vRZsVa3DO1gJ7q4VIwExD1NUFBXlc7w8JGnEoSd2TLxP3eFbh

Thanks so much, and looking forward to connecting.

Mack

**Comment by loveyou001** [comment]
[hi]( #433 )
